### PR TITLE
Pagy added

### PR DESCRIPTION
**Pull request by hammad-Ikhlaq-7vals**

### CODE REVIEW

**Review by hammad-Ikhlaq-7vals — COMMENTED** [review]
Please ignore lockable changes here. There is a separate PR for devise.

File: test/controllers/users_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (muhammad-basil) [inline test/controllers/users_controller_test.rb]: remove file

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (muhammad-basil) [inline app/assets/stylesheets/users.scss]: remove file

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (muhammad-basil) [inline app/helpers/users_helper.rb]: remove

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+include Pagy::Backend
+class UsersController < ApplicationController
+  def index
+    @pagy, @users = pagy(User.where(params[:company_id]),  items: 5)
```
Comment (muhammad-basil) [inline app/controllers/users_controller.rb:4]: why do we need both @pagy and @users?

Comment (hammad-Ikhlaq-7vals) [reply app/controllers/users_controller.rb:4]: I am using both in my index.html.erb file

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,3 @@
 module ApplicationHelper
+  include Pagy::Frontend
```
Comment (muhammad-basil) [inline app/helpers/application_helper.rb:2]: why do we need this?

File: db/schema.rb [hunk 6]
```diff
@@ -10,7 +10,20 @@
 #
```
Comment (muhammad-basil) [inline db/schema.rb]: should not be added in PR

Comment (hammad-Ikhlaq-7vals) [reply db/schema.rb]: schema.rb and database.yml files are included in .gitignore. But still they push.

File: app/helpers/application_helper.rb (hunk 5 above)
Comment (hammad-Ikhlaq-7vals) [reply app/helpers/application_helper.rb:2]: It is required for page_nav function in index.html.erb file

File: app/views/users/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<h1>Users</h1>
+
+  <% @users.each do |user| %>
+    <%= user.name %>
+    <%= user.email %>
+    <br>
+  <% end %>
+<%== pagy_nav(@pagy) %>
```
Comment (mujeeb-arshad) [inline app/views/users/index.html.erb]: what is the use of == ?

Comment (hammad-Ikhlaq-7vals) [reply app/views/users/index.html.erb]: It is assuming the incoming html is safe.
We can also write <%= pagy_nav(@pagy).html_safe %>

File: app/views/users/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<h1>Users</h1>
```
Comment (hafsabatool7) [inline app/views/users/index.html.erb]: Internationalization. Don't create a new key, use existing from shared.

Comment (hammad-Ikhlaq-7vals) [reply app/views/users/index.html.erb]: done

**Pull request merged**